CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.1] - 2023-07-04
+### Changed
+- Update dependencies, pin gear libs to `#946ac47` commit.
+
```

Comment:
We don't increase the version in `app` as it is a template for other programs rather than a standalone program.
```suggestion
```